src/backend/cuda/nvrtc/cache.cpp

```diff
@@ -378,6 +418,80 @@ string toString(const char *str) {
     return string(str);
 }
 
+template<>
+string toString(af_interp_type p) {
+    const char *retVal = NULL;
+#define CASE_STMT(v) \
+    case v: retVal = #v; break
+    switch (p) {
+        CASE_STMT(AF_INTERP_NEAREST);
+        CASE_STMT(AF_INTERP_LINEAR);
+        CASE_STMT(AF_INTERP_BILINEAR);
+        CASE_STMT(AF_INTERP_CUBIC);
+        CASE_STMT(AF_INTERP_LOWER);
+#if AF_API_VERSION >= 34
```

Comment:
You don't need api guards internally

Reply:
It is better to have it consistent with `af/defines.h` header.